api/test/wfl/unit/modules/covid_test.clj

```diff
@@ -0,0 +1,52 @@
+(ns wfl.unit.modules.covid-test
+  "Test the COVID workload module."
+  (:require [clojure.test        :refer :all]
+            [clojure.string      :as str]
+            [wfl.jdbc            :as jdbc]
+            [wfl.module.covid    :as covid]
+            [wfl.tools.fixtures  :as fixtures]
+            [wfl.tools.workloads :as workloads]
+            [wfl.wfl             :as wfl])
+  (:import [java.time OffsetDateTime]
+           [java.util UUID]))
+
+(use-fixtures :once fixtures/temporary-postgresql-database)
```

Comment:
likewise database usage generally means an integration test.

Reply:
OK.  I could probably mock more, but then we aren't really testing anything.